doc/porting/board_porting.rst

```diff
@@ -359,16 +418,13 @@ The following Kconfig extensions are available:
 
       grsource "Kconfig[12]"
 
-.. _zephyr-specific_kconfig_behavior_for_defaults:
-
-Zephyr-specific Kconfig behavior for defaults
-=============================================
+Old Zephyr Kconfig behavior for defaults
+========================================
 
-Zephyr uses a Kconfig patch that gives later ``default``\ s precedence over
-earlier ``default``\ s. This is a significant change from standard Kconfig
-behavior, which is to pick the first ``default`` with a satisfied condition.
+Older versions of Zephyr used a custom patch that made Kconfig prefer the last
```

Comment:
For (future) clarity, change to (verify the right version here):

    Versions of Zephyr prior to 1.13 used a custom ...

Reply:
Not sure what to put here, as I expect it to go in before the 1.13 release. I'll stall a bit.

Reply:
Updated it to say "prior to early August 2018 (during development of Zephyr 1.13), Zephyr used a custom patch..."

Is that okay?